components/formats-gpl/src/loci/formats/in/SlideBook6Reader.java

```diff
@@ -81,14 +82,14 @@
 	private static boolean libraryFound = false;
 
 	static {
-		String library_path = System.getProperty("java.library.path");
-
 		try {
 			// load JNI wrapper of SBReadFile.dll
-			System.loadLibrary("SlideBook6Reader");
-			libraryFound = true;
+			if (!libraryFound) {
+				libraryFound = NativeLibraryUtil.loadNativeLibrary(SlideBook6Reader.class, "SlideBook6Reader");
```

Comment:
This is throw an IOException rather than an UnsatisifiedLinkError:
```
WARNING: IOException creating DefaultJniExtractor
java.io.IOException: Couldn't find resource META-INF/lib/linux_64/ libSlideBook6Reader.so
        at org.scijava.nativelib.BaseJniExtractor.extractJni(BaseJniExtractor.java:144)
        at org.scijava.nativelib.NativeLibraryUtil.loadNativeLibrary(NativeLibraryUtil.java:265)
        at loci.formats.in.SlideBook6Reader.<clinit>(SlideBook6Reader.java:88)
```